src/main/java/seedu/address/Main.java

```diff
@@ -19,7 +22,23 @@
  * to be the entry point of the application, we avoid this issue.
  */
 public class Main {
+    private static Logger logger = LogsCenter.getLogger(Main.class);
+
+    /**
+     * Main entry point to the application.
+     */
     public static void main(String[] args) {
+        /*
+         * Since JavaFX 16, JavaFX runtime logs a warning if javafx.* modules are loaded from the classpath.
+         * This will result in a warning message whenever the application is launched.
+         * As modules are ignored when packed into a FAT Jar file, this warning will always be shown regardless
+         * of whether the application is using modules or not.
+         *
+         * This warning however, can be safely ignored. Thus, this log is used to inform the user that the warning
+         * can be ignored.
+         * https://github.com/openjdk/jfx/blob/master/doc-files/release-notes-16.md
+         */
```

Comment:
Original comment was OK. But rewrote to see if the flow can be refined further.
```suggestion
        /*
         * As per https://github.com/openjdk/jfx/blob/master/doc-files/release-notes-16.md
         * JavaFX 16 (or later) runtime logs a warning at startup if JavaFX classes are loaded from
         * the classpath instead of a module.
         * Our application does not use Java modules yet. Even if it did, modules are ignored when
         * packed into a FAT Jar file (as we do), which means this warning will persist even then.
         * The warning however, can be safely ignored. Thus, the following log informs
         * the user (if looking at the log output) that the said warning appearing in the log 
         * can be ignored.
         */
```